### ISSUE TITLE
Intro/backup files

### ISSUE THREAD

**Issue by donostiarra2** [karlheyes/icecast-kh#337]
Hi! I use Icecast KH with Centovacast. I'm trying the new master release, but if I set up any intro or backup file, it doesn't works neither. It's a bug? Thanks!

**Comment by scysys** [comment]
It´s definitely working! You have to make sure that you use the same channel, samplerate and bitrate for your intro / backup files. Also i dont think this is the right place for your centova related problems.

**Comment by donostiarra2** [comment]
Yes, of course, I have tried it correctly, without success. Although it's used with Centovacast, it simply manages the configuration file, so I think it's equally valid. I have checked that the configuration file used is correct.

**Comment by karlheyes** [comment]
That looks to be a validation check, to watch for consistencies. 2.4.4 does not attempt to check for those so won't complain.  It thinks there is an issue so that begs the question of what is in the file and what are the stream settings.  eg A 128k stream and a 32k intro will not match, although you say they are the same.   Why not post the intro file and provide the settings you have for the stream (either from the source or show the log level 4 entries for the stream detection at connection time).

karl

**Comment by donostiarra2** [comment]
Yes, the samplerate, channel number and bitrate it was exactly the same: 128kbps, 44100Hz, estereo. It was just a test to try this KH15 version, and I tried with the song I attach. Thanks.
[Ava Max - Torn.zip](https://github.com/karlheyes/icecast-kh/files/5458237/Ava.Max.-.Torn.zip)

**Comment by karlheyes** [comment]
I suspect its because the ID3 is large due to the png data in there. I'll look at fixing that case.

karl

**Comment by ale-rinaldi** [comment]
Hello,

I don't know if this can be of any help, but in a very similar situation I managed to solve the issue by placing the bitrate in the file name, between square brackets.

So for example the file name could be fallback[128].mp3

This solved the problem for me.

Hope it helps.

**Comment by simon1tan** [comment]
@ale-rinaldi Can you point to documentation on using these brackets?

**Comment by grassfedreeve** [comment]
Hey sorry to comment on such an old issue, unsure if it is better to create a new issues or simply comment here.

Testing on KH22 I am still seeing this issue as well. It seems like Karl was correct and this is showing on files with more data from album art. 

> I suspect its because the ID3 is large due to the png data in there. I'll look at fixing that case.

Any updates regarding this? Seems like stripping the file of it's album art will get it to play as a workaround.
